frontend/src/components/app/App.js

```diff
@@ -36,6 +37,7 @@ const App = () => {
             element={token ? <NewReview /> : <Navigate to="/login" />}
           />
           <Route path="/about" element={<AboutPage />} />
+          <Route path="/404" element={<NotFound />} />
```

Comment:
if you set the path to `*` then it will automatically take you here if you try to access any page that we haven't defined a route for

`<Route path="*" element={<NotFound />} />`